config/defaults.conf

```diff
@@ -172,6 +172,13 @@ patience = 5  // Patience in early stopping. Training will stop if performance d
               // all in patience + 1 validations.
 keep_all_checkpoints = 0  // If set, keep checkpoint files from every validation. Otherwise, keep
                           // only the best and (if different) most recent.
+delete_checkpoints_when_done = 0  // If set, delete *all* saved checkpoints when the run is
```

Comment:
is it really space consuming to keep just the best (or last N) checkpoints around? Are they 1GB each?